src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
```

Comment:
You're using a fixed size for the `.star` class, which is acceptable. Just make sure that this fixed size aligns with the design requirements and doesn't cause issues with responsiveness or scaling.